### DISCUSSION TITLE
Hello, may I ask when this version 4.2 can be released approximately?

### DISCUSSION THREAD

**Discussion by 1853582** [tuneinsight/lattigo#392] [Q&A] [answered]
Sorry to bother you, I see that the new version integrates many new features. I would like to inquire when version 4.2 can be released?

**Comment by ChristianMct** [comment, marked as answer]
Hi @1853582,

The v4.2.0 release indeed integrates many new features. We are about done implementing these features, and we have begun the code review.

Without making any promise, I'd say that a release at the end of August/beginning of September is realistic.
